src/app/components/EditorInterface.tsx

```diff
@@ -199,22 +188,25 @@ const EditorInterface = ({ settings, token }: IEditorInterface) => {
       })
     }
   }, [
-    appState?.appState.selectedClient,
     appState?.appState.selectedClientCompanyName,
+    appState?.appState.selectedClient,
+    appState?.appState.notifications,
   ])
 
   useEffect(() => {
-    if (!appState?.appState.readOnly) {
-      appState?.setOriginalTemplate(editor?.getHTML() as string)
-    }
-  }, [editor?.getHTML(), appState?.appState.readOnly])
-
-  useEffect(() => {
-    if (editor && appState?.appState.settings?.content.includes(defaultState)) {
+    // ! This will break someday
```

Comment:
please rephrase the comment stating actual issue and actions needed to be taken

Reply:
i have removed this comment